app/Http/Controllers/ReportsController.php

```
@@ -419,23 +354,181 @@ function splitAmountInTwo($amount) {
+                    'journal_date' => $as_of->format('Y-m-d'),
+                    'branch_id' => 4,
+                    'book_id' => $journalEntry::DEPRECIATION_BOOK,
+                    'source' => $journalEntry::DEPRECIATION_SOURCE,
+                    'status' => $journalEntry::STATUS_POSTED,
+                    'remarks' => 'Representing Month End Schedule As of ' . $as_of . '-' . $accountName,
+                    'amount' => $subsidiaryCategory->sub_cat_code === SubsidiaryCategory::INSUR_ADD
+                        ? $totalPrepaidExpense
+                        : $totalMonthlyAmort,
+                ]);
+                $journalEntry->details()->createMany($journalDetails);
+            });
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'postDepreciation'. (Clean Code Rules)

(StaticAccess)